mreg_cli/host.py

```diff
@@ -1012,7 +1012,7 @@ def _ip_remove(args, ipversion):
     # Check that ip belongs to host
     info = host_info_by_name(args.name)
     for rec in info["ipaddresses"]:
-        if rec["ipaddress"] == args.ip:
+        if rec["ipaddress"].lower() == args.ip.lower():
```

Comment:
I'm pretty sure mreg itself is strictly lowercase. Maybe remove the first `lower()` since it's unnecessary?

Reply:
Added a new commit that does this.